LayoutTests/platform/ios/TestExpectations

```diff
@@ -32,6 +32,7 @@ fast/events/ios/ipad [ Skip ]
 fast/forms/ios [ Pass ]
 fast/forms/ios/ipad [ Skip ]
 fast/forms/switch [ Pass ]
+fast/events/focus-keyboard-event-crash.html [ Pass ]
```

Comment:
No, you need to say either `Skip` or `WontFix`. Also, this isn't the right place to insert this expectation. Move to more towards the end of the file.

Reply:
Updated.